frontend/src/bundles/common/helpers/convert-height/convert-height-to-mm.ts

```diff
@@ -0,0 +1,14 @@
+const convertHeightToMillimeters = (
+    height: string | number | null,
+): number | null => {
+    if (height === null) {
+        return null;
+    }
+
+    const heightInCm =
+        typeof height === 'string' ? Number.parseFloat(height) : height;
+
```

Comment:
duplicating logic, maybe we can somehow make a one helper instead of three

Reply:
```
import { KG_TO_GRAMS_CONVERSION_FACTOR, MM_TO_CM_CONVERSION_FACTOR, ROUNDING_FACTOR } from './constants/constants.js';

const convertToTargetUnit = (
  value: string | number | null,
  conversionFactor: number,
  roundingFactor: number = 1
): number | null => {
  if (!value) {
    return null;
  }

  const numericValue = typeof value === 'string' ? Number.parseFloat(value) : value;

  return Number.isNaN(numericValue)
    ? null
    : Math.round(numericValue * conversionFactor * roundingFactor) / roundingFactor;
};

const convertWeightToGrams = (weight: string | number | null): number | null => {
  return convertToTargetUnit(weight, KG_TO_GRAMS_CONVERSION_FACTOR, ROUNDING_FACTOR);
};

const convertHeightToMillimeters = (height: string | number | null): number | null => {
  return convertToTargetUnit(height, 1, MM_TO_CM_CONVERSION_FACTOR);
};

const convertHeightToCentimeters = (height: string | number | null): number | null => {
  return convertToTargetUnit(height, MM_TO_CM_CONVERSION_FACTOR, MM_TO_CM_CONVERSION_FACTOR);
};

export { convertWeightToGrams, convertHeightToMillimeters, convertHeightToCentimeters };
```
You mean something like this? I don't know how it will be better to make

Reply:
Yeah, I thinks thats better